crates/gwt/web/vendor/xterm/xterm.css

```
@@ -0,0 +1,285 @@
+.xterm .xterm-scrollable-element > .scrollbar > .scra {
+	cursor: pointer;
+	font-size: 11px !important;
+}
+
+.xterm .xterm-scrollable-element > .visible {
+	opacity: 1;
+
+	/* Background rule added for IE9 - to allow clicks on dom node */
+	background:rgba(0,0,0,0);
+
+	transition: opacity 100ms linear;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)